### ISSUE TITLE
[1.10.2]Arcane Compendium showing everything

### ISSUE THREAD

**Issue by ImperfMage** [Mithion/ArsMagica2#1620]
So I just made the Arcane Compendium and all the entries were unlocked. So I went in and check the config and it says 

# Set this to false to have the compendium show everything, and not unlock as you go.
    B:"Staged Compendium"=true

I know it's something small , but I really like the staged aspect of it. If possible please fix as soon as possible. 

Ars Magica currently installed is: Ars Magica 2 v1.5.0-18
Current forge is : forge-12.18.3.2281

**Comment by renadi** [comment]
Mithion is no longer developing Ars Magica, Growlithe is the dev of 1.10
versions and on, 1.7.10 is done seeing releases I think.

**Comment by ImperfMage** [comment]
@renadi Yes I am aware of the 1.7.10 being done with release , I am talking about the 1.10.2 version

**Comment by renadi** [comment]
https://github.com/Growlith1223/ArsMagica2/issues

**Comment by ImperfMage** [comment]
oh sorry didn't know there was a different one , thanks
